src/support/string.cpp

```diff
@@ -432,4 +432,60 @@ bool isUTF8(std::string_view str) {
   return true;
 }
 
+std::vector<char> unescapeJSON(const char* str) {
```

Comment:
It would be good to put the output encoding in the function name to avoid ambiguity. I guess we want WTF-8 to match the encoding of the imported string constants. We'll probably want to do WTF-8 validation here as well, so it might be worth having the function return a bool already.

Reply:
Renamed.

I'm not sure we need to return a bool - can just fatally error? If we ever need to handle such things we can change it, but I doubt we will.